commit 8ba69737d0f7a1522ac55a4d5d2613347532d411
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Jan 15 16:32:33 2019 +0000

    staging: bcm2835-camera: Set the field value within each buffer
    
    Fixes a v4l2-compliance failure
    v4l2-test-buffers.cpp(415): g_field() == V4L2_FIELD_ANY
    
    The driver only ever produces progresive frames, so field should
    always be set to V4L2_FIELD_NONE.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
index dbdc4e84f3b1..a8f3ce5afb51 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
@@ -443,6 +443,7 @@ static void buffer_cb(struct vchiq_mmal_instance *instance,
 	}
 	dev->capture.last_timestamp = buf->vb.vb2_buf.timestamp;
 	buf->vb.sequence = dev->capture.sequence++;
+	buf->vb.field = V4L2_FIELD_NONE;
 
 	vb2_set_plane_payload(&buf->vb.vb2_buf, 0, mmal_buf->length);
 	if (mmal_buf->mmal_flags & MMAL_BUFFER_HEADER_FLAG_KEYFRAME)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1470_8ba69737d0f7a1522ac55a4d5d2613347532d411.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1470_8ba69737d0f7a1522ac55a4d5d2613347532d411.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
